src/playground/Pages/Tabs/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { Header } from "@/components/Blocks/Header"
+import { Breadcrumb } from "@/components/Navigation/Breadcrumb"
+import {
+  Tabs as TabsComponent,
+  TabsContent,
+  TabsList,
+  TabsTrigger,
+} from "@/components/Navigation/Tabs"
```

Comment:
I feel we'll enventually have to create different kinds of tabs or at least differentiate the "navigational" ones (only used in pages) from other kinds of tabs (forms, etc). But this is OK for now.

Reply:
Agreed